openunreid/apis/test.py

```diff
@@ -207,3 +202,31 @@ def infer_gan(
     print(f"\n{sep} Finished translating {sep}\n")
 
     return
+
+
+@torch.no_grad()
+def final_test(cfg, model, cmc_topk=(1, 5, 10)):
```

Comment:
The module name "final_test" is somehow confusing. Maybe you could modify the original "test_reid" into "test_reid_once" and "final_test" into "test_reid".